core/src/test/scala/org/apache/spark/sql/delta/DeltaErrorsSuite.scala

```diff
@@ -47,7 +47,7 @@ import org.apache.spark.sql.catalyst.expressions.{AttributeReference, ExprId, Le
 import org.apache.spark.sql.catalyst.expressions.Uuid
 import org.apache.spark.sql.catalyst.parser.CatalystSqlParser
 import org.apache.spark.sql.connector.catalog.Identifier
-import org.apache.spark.sql.internal.SQLConf
+import org.apache.spark.sql.internal.{SQLConf, StaticSQLConf}
```

Comment:
Looks like the changes in this file are not needed?

Reply:
will revert back the changes.